src/side-navigation/side-navigation.scss

```diff
@@ -170,3 +171,71 @@
     }
   }
 }
+
+@mixin iui-side-navigation-submenu {
+  display: none;
+  visibility: hidden;
+  opacity: 0;
+  width: 0;
+  height: 100%;
+  box-sizing: border-box;
+  padding: 0 $iui-sm;
+  overflow-x: hidden;
+  overflow-y: overlay;
+  resize: horizontal;
+  @include themed {
+    background-color: t(iui-color-background-1);
+    border-right: 1px solid t(iui-color-background-5);
+  }
+
+  &.iui-side-navigation-submenu-visible {
```

Comment:
I think this class is not needed and some rules can be moved to the submenu class and others moved to HTML.

Reply:
I was thinking the same thing

Reply:
I think it would be better to toggle the HTML itself (i.e. remove from DOM when collapsed) than toggle visibility through CSS.

Reply:
Ok, I've moved this out of the CSS and into the HTML.  This class handles making it visible (display, visibility, & opacity) but also included some width, min, & max.  Will those also be handled in React?

Reply:
The width stuff can moved into the general submenu mixin. And the visibility part can be handled in React using transition-group classes.

Reply:
Oh you moved the CSS verbatim, and I see we also have some more rules
![image](https://user-images.githubusercontent.com/9084735/134380211-bbaada48-0447-41bf-b133-7dade2f1e7d5.png)

Those will all need to be removed. And instead, we'll need to add some JS for show/hide (similar to `expandable-block.html`). The animation will not work in the demo html, but that's ok.

The width/min/max stuff can moved into the general submenu mixin. And the enter/exit animation can be handled in React using transition-group classes.

Reply:
The submenu demo has the submenu stuck in the open state, so none of the display, visibility, opacity are present anymore.